.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,34 @@
 - [ ] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
 
 I am:
-  - [ ] Suggesting edits to an existing Topic page
-  - [ ] Curating a new Topic page
+  - [ ] Suggesting edits to an existing topic or collection
+  - [ ] Curating a new topic or collection
 
-***********EDITING AN EXISTING TOPIC PAGE************
+***********EDITING AN EXISTING TOPIC OR COLLECTION************
 
-I'm suggesting these edits to an existing topic:
+I'm suggesting these edits to an existing topic or collection:
 - [ ] Image (and my file is `*.png`, square, dimensions 288x288)
 - [ ] Content (and my changes are in `index.md`)
 
 Please explain why these changes are necessary:
 
 
-************CURATING A NEW TOPIC PAGE************
+************CURATING A NEW TOPIC************
 
 - [ ] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
 - [ ] My folder contains a `*.png` image (if applicable) and `index.md`
 - [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs
 
-Please explain why you think this Topic page should be curated:
+Please explain why you think this topic should be curated:
+
+
+************CURATING A NEW COLLECTION************
```

Comment:
Will having the split between curating a topic versus a collection be confusing for contributors? Maybe we should lean on [multiple PR templates](https://github.com/blog/2495-multiple-issue-and-pull-request-templates) and have a separate one for topics versus collections.

Reply:
✅ Done